### PR TITLE
improve MissingPolicyException message for Query instances

### PR DESCRIPTION
**Pull request by steinkel**

### CODE REVIEW

File: src/Policy/Exception/MissingPolicyException.php [hunk 1]
```diff
@@ -37,7 +38,20 @@ class MissingPolicyException extends Exception
...
+            $resourceClass = get_class($resource);
+            if (
+                method_exists($resource, 'getRepository') &&
+                $resource->getRepository() &&
+                $resource->getRepository() instanceof RepositoryInterface
+            ) {
+                $repositoryClass = get_class($resource->getRepository());
+                $resource = sprintf($this->_messageTemplate, $resourceClass);
+                $queryMessage = ' This resource looks like a Query, if you are using the OrmResolver, ' .
+                    'you might need to create a new policy class for your %s class in `src/Policy/`';
```
Comment (othercorey) [inline src/Policy/Exception/MissingPolicyException.php:50]:
```suggestion
                $queryMessage = ' This resource looks like a Query. If you are using the OrmResolver, ' .
                    'you might need to create a new policy class for your %s class in `src/Policy/`.';
```

**Comment by othercorey** [comment]
@markstory can you review this?

File: src/Policy/Exception/MissingPolicyException.php [hunk 2]
```diff
@@ -37,7 +38,20 @@ class MissingPolicyException extends Exception
...
+            $resourceClass = get_class($resource);
+            if (
+                method_exists($resource, 'getRepository') &&
+                $resource->getRepository() &&
+                $resource->getRepository() instanceof RepositoryInterface
+            ) {
+                $repositoryClass = get_class($resource->getRepository());
+                $resource = sprintf($this->_messageTemplate, $resourceClass);
+                $queryMessage = ' This resource looks like a Query. If you are using OrmResolver, ' .
+                    'you might need to create a new policy class for your %s class in `src/Policy/`.';
```
Comment (markstory) [inline src/Policy/Exception/MissingPolicyException.php:50]:
```suggestion
                $queryMessage = ' This resource looks like a `Query`. If you are using `OrmResolver`, ' .
                    'you should create a new policy class for your `%s` class in `src/Policy/`.';
```

**Review by markstory — COMMENTED** [review]
Some minor formatting fixes, but otherwise looks good to me.

**Comment by othercorey** [comment]
@steinkel 2.next is going to be 3.0 release so this should go into master.

**Comment by steinkel** [comment]
@othercorey rebased/fixed for 2.next thanks!

**Pull request merged**